models/aeif_cond_alpha.h

```diff
@@ -103,7 +103,15 @@ and
 For the reference implementation of this model, see
 `aeif_models_implementation <../model_details/aeif_models_implementation.ipynb>`_ notebook.
 
-See also [1]_.
+
+.. note::
+
+    The default refractory period for ``aeif`` models is zero, consistent with the model definition in
+    Brette & Gerstner [1]_.
+    This means if the refractory period is zero, the ``aeif`` neurons can fire multiple spikes in a single time step,
+    which can lead to exploding spike numbers and extreme slow-down of simulations.
```

Comment:
Based on Charl's comment, I suggest the following (applies to all other models as well):
```suggestion
    The default refractory period for ``aeif`` models is zero, consistent with the model definition in
    Brette & Gerstner [1]_.  Thus, an ``aeif`` neuron with default parameters can fire multiple spikes in a single time step,
    which can lead to exploding spike numbers and extreme slow-down of simulations.
```

Reply:
changes applied to all models